### ISSUE TITLE
Detection for used coffee pad

### ISSUE THREAD

**Issue by phillipthelen** [ThomDietrich/SenseoWifi#67]
Im not sure if this is something that was ever considered. But it would be very useful if the coffee machine could have some knowledge about whether or not a new coffee pad is in it. If there is a sensor on the lid, the machine would have the data to know if the lid was opened since the last brewing. Or also prevent brewing if the lid is open. This would be useful for cases where you forget to put in a new pad and have the coffee machine brew coffee automatically in the morning

**Comment by ThomDietrich** [comment]
I agree. This would be a good addition. You could tinker with a reed switch under the case and a small magnet in the lid. No leaking holes needed... Let us know what you got to!
